tests/examples/pacer/attachment_pages/mad_0951290550.json

```diff
@@ -1,7 +1,7 @@
 {
   "attachments": [
     {
-      "attachment_number": 2,
+      "attachment_number": 1,
```

Comment:
Yikes!

Reply:
yeah...I bet we have quite a few issues like this